### PR TITLE
Re-structure the repo and added poetry related files

### PR DESCRIPTION
**Pull request by leon332157**
I have tested this so all of the import works. Not sure entirely if this works on replit, needs more testing.

### CODE REVIEW

**Comment by Codemonkey51** [comment]
I don't get how just moving almost all the files to an src folder. I agree this repo needs to be rearranged but it should be done differently, like grouping server files into a module, same with discord bot

**Comment by leon332157** [comment]
I have tried to do so, with the file structure being very messy, some import does not work. I think I will probably condense the file a little more but having multiple folders doesn't work well with Python imports.

**Comment by Codemonkey51** [comment]
> I have tried to do so, with the file structure being very messy, some import does not work. I think I will probably condense the file a little more but having multiple folders doesn't work well with Python imports.

I can try to do my idea locally then if it works pr it :/

**Comment by leon332157** [comment]
@Codemonkey51 Have you worked on anything. What we have right now seems fine. I think it's best if we merge this to the master then start from there since we have the plans as to what is going to happen.

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -19,14 +19,11 @@ jobs:
...
+        if [ -f pyproject.toml ]; then poetry install; fi
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --ignore=W191,E701
-    - name: Test with pytest
-      run: |
-        pytest
```
Comment (Codemonkey51) [inline .github/workflows/python-app.yml:32]: Un-remove pytest please, it is needed for the test suite

**Review by Codemonkey51 — DISMISSED** [review]

**Review by mat-1 — CHANGES_REQUESTED** [review]
spaces are cringe, use tabs pls

**Review by mat-1 — CHANGES_REQUESTED** [review]
ok double quotes are cringe as well, use single quotes pls

**Comment by leon332157** [comment]
Since there's a bug with black, I'm going to keep it but not enforcing it.

**Comment by leon332157** [comment]
nice now pytest technically work. Now it's time to clean up the code more.

**Review by mat-1 — APPROVED** [review]
Looks good!

**Comment by leon332157** [comment]
???

**Review by Codemonkey51 — APPROVED** [review]
mat says good so i say good

**Pull request merged**